app/components/file-contents-card.ts

```diff
@@ -9,9 +12,11 @@ interface Signature {
     code: string;
     language: string;
     filename: string;
-    headerTooltipText?: string; // If collapsible, use this to provide a tooltip for the header
```

Comment:
Would keep this comment in, the property name doesn't fully convey why one would want to set this

Reply:
Should I rename the argument to `collapsedHeaderTooltipText` instead? Sounds like a more proper name.

Reply:
Wrote proper JSDocs to the whole signature of `FileContentsCard`